console/react/src/details/createTablePage.test.js

```diff
@@ -18,7 +18,7 @@ under the License.
 */
 
 import React from "react";
-import { render, fireEvent, waitForElement } from "@testing-library/react";
+import { render, fireEvent } from "@testing-library/react";
```

Comment:
This is for `21:29  warning  'waitForElement' is defined but never used  no-unused-vars`